GitExtUtils/GitUI/TaskExtensions.cs

```diff
@@ -1,200 +0,0 @@
-﻿using System;
```

Comment:
Were none of these API used, or are they now provided by the new version Microsoft.VisualStudio.Threading?

Reply:
They were added in the updated version